### PR TITLE
feat(TBD-7402): support multiple s3 buckets

### PR DESCRIPTION
**Pull request by hliashchuk**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.dbr350/src/main/java/org/talend/hadoop/distribution/dbr350/DBR350Distribution.java [hunk 1]
```diff
@@ -233,12 +233,12 @@ public boolean doSupportS3V4() {
 
     @Override
     public boolean useOldAWSAPI() {
-        return false;
+        return true;
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.dbr350/src/main/java/org/talend/hadoop/distribution/dbr350/DBR350Distribution.java:236]: no need to modify this plugin as the distro is EoS, but why not

File: main/plugins/org.talend.hadoop.distribution.dbr550/src/main/java/org/talend/hadoop/distribution/dbr550/DBR550Distribution.java [hunk 2]
```diff
@@ -233,7 +233,7 @@ public boolean doSupportS3V4() {
 
     @Override
     public boolean useOldAWSAPI() {
-        return false;
+        return true;
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.dbr550/src/main/java/org/talend/hadoop/distribution/dbr550/DBR550Distribution.java:236]: It's unclear to me why we need to use the old API

Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.dbr550/src/main/java/org/talend/hadoop/distribution/dbr550/DBR550Distribution.java:236]: same question. It's been required by old s3 libs and it's replaced with s3.accessKey and Secret properties. It's just responsible for adding key/secret to aws bucket

Comment (hliashchuk) [reply main/plugins/org.talend.hadoop.distribution.dbr550/src/main/java/org/talend/hadoop/distribution/dbr550/DBR550Distribution.java:236]: https://docs.databricks.com/data/data-sources/aws/amazon-s3.html
Databricks doesn't support s3a properties to configure accessKey per bucket, so we need to have URI in format s3n://access:secret@bucket to configure per bucket credentials

**Review by CarlosTalend — APPROVED** [review]

**Review by sponomarova — APPROVED** [review]
lgtm

**Pull request merged**